src/runtime/manifest.ts

```diff
@@ -145,6 +151,9 @@ export class Manifest {
     return this._id;
   }
   get storageProviderFactory() {
```

Comment:
can this be private by any chance?

Reply:
Unfortunately not quite yet. It's used in one other place.